### PR TITLE
Remove server code from chill-dashboard

### PR DESCRIPTION
**Pull request by badyakarprabin**
* Remove server code

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -10,17 +10,15 @@ dotenv.config();
 module.exports = {
   entry: [
     'react-hot-loader/patch',
-    'webpack-dev-server/client?http://localhost:8181', // WebpackDevServer host and port
```
Comment (mesaugat) [inline webpack.config.js]: Any reason for removing this?

File: package.json [hunk 2]
```diff
@@ -3,19 +3,11 @@
   "version": "1.0.0-alpha.1",
   "description": "Chill Dashboard",
   "scripts": {
-    "start": "run-p start:app start:server",
-    "start:app": "webpack-dev-server",
-    "start:server": "nodemon --watch .env --watch src/server --exec babel-node src/server --source-maps",
+    "start": "webpack-dev-server",
     "clean": "rimraf dist",
-    "build": "NODE_ENV=production run-s lint clean build:all",
```
Comment (kabirbaidhya) [inline package.json]: Don't remove it. Just keeping it as `NODE_ENV=production run-s lint clean build:app` would be fine.

File: package.json [hunk 3]
```diff
@@ -30,38 +22,17 @@
...
   "dependencies": {
     "axios": "^0.15.3",
-    "body-parser": "^1.15.2",
-    "bookshelf": "^0.10.2",
-    "bookshelf-camelcase": "^1.1.4",
-    "boom": "^4.2.0",
     "bootstrap": "^3.3.7",
-    "compression": "^1.6.2",
-    "cors": "^2.8.1",
     "dotenv": "^4.0.0",
```
Comment (kabirbaidhya) [inline package.json]: move this to `devDependencies`.

File: package.json [hunk 4]
```diff
@@ -86,16 +57,12 @@
     "identity-obj-proxy": "^3.0.0",
     "istanbul": "1.1.0-alpha.1",
```
Comment (mesaugat) [inline package.json]: We probably don't need this as well.

File: package.json [hunk 5]
```diff
@@ -30,38 +22,17 @@
...
-    "winston": "^2.3.0",
-    "winston-daily-rotate-file": "^1.4.0"
+    "react-router-dom": "^4.0.0"
   },
   "devDependencies": {
     "autoprefixer": "^6.7.2",
-    "babel-cli": "^6.18.0",
-    "babel-core": "^6.22.1",
-    "babel-eslint": "^7.1.1",
-    "babel-jest": "^19.0.0",
```
Comment (mesaugat) [inline package.json]: `Jest` needs this.

**Comment by kabirbaidhya** [comment]
Let's make our eslint rules consistent. Can we copy the `.eslintrc.yml` file from the chill repo here? https://github.com/leapfrogtechnology/chill/blob/master/.eslintrc.yml

File: package.json [hunk 6]
```diff
@@ -30,38 +22,17 @@
...
     "babel-loader": "^6.2.10",
     "babel-plugin-istanbul": "^4.1.1",
```
Comment (mesaugat) [inline package.json]: Not required

File: package.json [hunk 7]
```diff
@@ -86,16 +57,12 @@
...
     "node-sass": "^4.5.0",
-    "nodemon": "^1.11.0",
     "npm-run-all": "^4.0.2",
     "postcss-loader": "^1.3.3",
     "react-addons-test-utils": "^15.4.2",
     "react-hot-loader": "3.0.0-beta.6",
-    "rimraf": "^2.5.4",
     "sass-loader": "^6.0.2",
-    "semver": "^5.3.0",
     "sinon": "^2.1.0",
```
Comment (mesaugat) [inline package.json]: `sinon` and `sinon-chai` probably are not required as well.

File: package.json [hunk 8]
```diff
@@ -76,28 +49,22 @@
...
-    "istanbul": "1.1.0-alpha.1",
     "jest": "^19.0.2",
-    "mocha": "^3.2.0",
     "node-sass": "^4.5.0",
-    "nodemon": "^1.11.0",
     "npm-run-all": "^4.0.2",
     "postcss-loader": "^1.3.3",
     "react-addons-test-utils": "^15.4.2",
     "react-hot-loader": "3.0.0-beta.6",
-    "rimraf": "^2.5.4",
```
Comment (kabirbaidhya) [inline package.json]: the npm script `"clean": "rimraf dist",` needs this.

File: package.json [hunk 9]
```diff
@@ -30,41 +23,21 @@
...
-    "bookshelf-camelcase": "^1.1.4",
-    "boom": "^4.2.0",
     "bootstrap": "^3.3.7",
-    "compression": "^1.6.2",
-    "cors": "^2.8.1",
     "dotenv": "^4.0.0",
-    "express": "^4.14.1",
     "font-awesome": "^4.7.0",
-    "helmet": "^3.2.0",
+    "fs": "^0.0.1-security",
```
Comment (kabirbaidhya) [inline package.json]: No need, it's a built-in module.

Comment (mesaugat) [inline package.json]: 💯

File: package.json [hunk 10]
```diff
@@ -76,28 +48,23 @@
...
     "postcss-loader": "^1.3.3",
     "react-addons-test-utils": "^15.4.2",
     "react-hot-loader": "3.0.0-beta.6",
     "rimraf": "^2.5.4",
     "sass-loader": "^6.0.2",
-    "semver": "^5.3.0",
-    "sinon": "^2.1.0",
-    "sinon-chai": "^2.8.0",
     "style-loader": "^0.16.1",
     "supertest": "^2.0.1",
```
Comment (mesaugat) [inline package.json]: Remove

File: package.json [hunk 11]
```diff
@@ -76,28 +48,23 @@
     "chai": "^3.5.0",
```
Comment (mesaugat) [inline package.json]: Remove

File: package.json [hunk 12]
```diff
@@ -76,28 +48,23 @@
...
     "codecov": "^2.1.0",
     "css-loader": "^0.28.0",
+    "compression": "^1.6.2",
```
Comment (mesaugat) [inline package.json]: Where do we need this? Remove.

File: package.json [hunk 13]
```diff
@@ -76,28 +48,23 @@
...
+    "cors": "^2.8.1",
```
Comment (mesaugat) [inline package.json]: And this?

File: package.json [hunk 14]
```diff
@@ -76,28 +48,23 @@
...
     "enzyme": "^2.7.1",
     "eslint": "^3.19.0",
     "eslint-plugin-react": "^6.10.0",
     "extract-text-webpack-plugin": "^2.0.0",
     "file-loader": "^0.11.1",
     "html-webpack-plugin": "^2.28.0",
     "http-proxy-middleware": "^0.17.3",
```
Comment (mesaugat) [inline package.json]: And this?

Comment (kabirbaidhya) [inline package.json]: LOL

Comment (romitamgai) [inline package.json]: :D

File: src/components/dashboard/MainWrapper.js [hunk 15]
```diff
@@ -28,4 +29,9 @@ const MainWrapper = (props) => (
   </div>
 );
 
+MainWrapper.propTypes = {
+  totalRunning: PropTypes.string,
+  totalStopped: PropTypes.string,
+  services: PropTypes.array
```
Comment (mesaugat) [inline src/components/dashboard/MainWrapper.js]: ↔️

**Comment by mesaugat** [comment]
I'd suggest everyone to write commits based on what you modified in the code and not `Fix PR comments` or something like this. Git commit is supposed to be a realization of what that commit did to the existing code and not anything else.

**Comment by kabirbaidhya** [comment]
Also, I don't think we need to prefix the branch name in the commits. We aren't using smart commits so it doesn't make sense here. @leapfrogtechnology/the-chill-team 
image

**Pull request merged**